### PR TITLE
fix(stepper): emits `calciteStepperItemChange` event when switched to first step

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #8397 

## Summary

Emits `calciteStepperItemChange` event when switched to the first enabled step in single view layout.

### CODE REVIEW

**Review by alisonailea — APPROVED** [review]

File: packages/calcite-components/src/components/stepper/stepper.tsx [hunk 1]
```diff
@@ -506,7 +504,6 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     }
 
     if (
-      this.currentActivePosition &&
```
Comment (driskull) [inline packages/calcite-components/src/components/stepper/stepper.tsx:509]: should this maybe just check the type? Something like...

- `this.currentActivePosition != null`?
- `typeof this.currentActivePostion === "number"`?

**Review by driskull — APPROVED** [review]
👍

File: packages/calcite-components/src/components/stepper/stepper.tsx (hunk 1 above)
Comment (anveshmekala) [reply packages/calcite-components/src/components/stepper/stepper.tsx:509]: Good catch, this made me realize we need that condition to support when all the item's are disabled. Will get that added.

Comment (driskull) [reply packages/calcite-components/src/components/stepper/stepper.tsx:509]: cool. yeah im guessing the issue was that the currentActivePostion was `0` which is a valid index number but causing this if statement to not get entered.

Comment (anveshmekala) [reply packages/calcite-components/src/components/stepper/stepper.tsx:509]: yup.

**Pull request merged**